data/src/main/java/me/fontys/semester4/data/entity/PostalcodePart.java

```diff
@@ -14,6 +14,15 @@
     private static final long serialVersionUID = 1L;
 
     @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "hilopooled")
+    @GenericGenerator(name = "hilopooled", strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator",
+            parameters = {
+                    @Parameter(name = "sequence_name", value = "hilo_sequence"),
```

Comment:
What does this do

Reply:
Using `@GeneratedValue` with type `IDENTIFY` should be enough

Reply:
If you want to wait 1 hour for the import sure, but that's not acceptable. The information for postgresql is clear here:
https://dzone.com/articles/50-best-performance-practices-for-hibernate-5-amp
item 20.
You have to understand that importing 600 000 records requires performance fine tuning.